workshop_operations/post_workshop_email_template.md

```diff
@@ -16,13 +17,13 @@ FIXME add link
 
 FIXME add link
```

Comment:
Which link should I add?

Reply:
None! It's just a template, the FIXME will be fixed when the template is copied into the Pindena system. Nothing to fix, just "approve" of the changes I made to the previous text :)